collect_app/src/test/java/org/odk/collect/android/widgets/BooleanWidgetTest.java

```diff
@@ -79,4 +79,8 @@ public void changingTheCheckboxShouldChangeTheAnswer() {
         assertTrue(widget.isChecked());
         assertTrue((Boolean) widget.getAnswer().getValue());
     }
+
+    @Test
+    public void usingReadOnlyOptionShouldMakeAllClickableElementsDisabled() {
```

Comment:
Is there just not anything to test here? If so I think we should just have a comment in the method body.

Reply:
This widget is unused, I tried to use it (it works like TriggerWidget) but it crashes. @lognaturel can we get rid of it at all?

Reply:
Ah that makes sense!

Reply:
Yes, I think we should get rid of it! At some point we explored what it would take to have a more true boolean type but we ran into semantics issues and I guess we never fully backed out.

Reply:
Ok so I removed it.